pay-api/src/pay_api/resources/__init__.py

```diff
@@ -80,4 +81,5 @@
 API.add_namespace(INVOICE_RECEIPT_API, path='/payment-requests/<int:payment_id>')
 API.add_namespace(ACCOUNTS_API, path='/accounts')
 API.add_namespace(ACCOUNT_STATEMENTS_API, path='/accounts/<string:account_id>/statements')
+API.add_namespace(ACCOUNT_STATEMENTS_SETTINGS_API, path='/accounts/<string:account_id>/statements-settings')
```

Comment:
/statements/settings would have been better I think, just a suggestion.

Reply:
yeah..was double minded about it... thanks for the suggestion